src/index.html

```diff
@@ -1,13 +1,440 @@
+          >
+            <h2 class="
+              move-free__title
+              section-title
+              section-title--align-left
+            ">
+              Move Free
+            </h2>
+
+            <p class="move-free__content">
+              Shift your ride, not gears. Find the fastest way to move in the city as the bike adapts intuitively to power the speed you need.
+            </p>
+          </section>
+
+          <section
+            class="compare"
+            id="compare-bikes"
+          >
+            <h2 class="section-title">Compare Bikes</h2>
+
+            <div class="compare__products">
+              <article class="product compare__product">
+                <a href="#">
+                  <img
+                    src="./images/products/Screenshot 2022-01-14 at 11.12 1.png"
+                    alt="Sporty 4"
+                    class="product__photo"
+                  >
+                </a>
+
+                <h3 class="product__title">Sporty 4</h3>
+                <p class="product__description">
+                  The iconic frame brought to a new performance height as a sporty, active ride.
+                </p>
+                <p class="product__price">
+                  $ 2 590
+                </p>
+              </article>
+
+              <article class="product compare__product">
+                <a href="#">
+                  <img
+                    src="./images/products/Screenshot 2022-01-14 at 11.12 2.png"
+                    alt="Ride in town ST"
+                    class="product__photo"
+                  >
+                </a>
+                <h3 class="product__title">Ride in town ST</h3>
+                <p class="product__description">
+                  An open frame for an upright riding position as the most comfortable ride in town.
+                </p>
+                <p class="product__price">
+                  $ 2 590
+                </p>
+              </article>
+
+              <article class="product compare__product">
+                <a href="#">
+                  <img
+                    src="./images/products/Screenshot 2022-01-14 at 11.12 3.png"
+                    alt="Agile ride 3"
+                    class="product__photo"
+                  >
+                </a>
+                <h3 class="product__title">Agile ride 3</h3>
+                <p class="product__description">
+                  The iconic frame brought to a new performance height as a sporty, active ride.
+                </p>
+                <p class="product__price">
+                  $ 2 090
+                </p>
+              </article>
+            </div>
+          </section>
+
+          <section
+            class="details"
+            id="details"
+          >
+            <h2 class="section-title">The Details</h2>
+
+            <div class="details__wrapper">
+              <article class="detail">
+                <div class="detail__photos">
+                  <a
```

Comment:
It's really necessary to have link here? It's just a block with information about details, and images are just illustration to this information. Not links.